### PR TITLE
docs: added move notice

### PR DESCRIPTION
**Pull request by BenjaminHalko**
We should let people know that development has moved to the ReVanced Manager repo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# ⚠️ MOVING NOTICE ⚠️
+
+All future development on the ReVanced Manager Compose has been moved to the [revanced-manager](https://github.com/ReVanced/revanced-manager) repository.
```
Comment (Axelen123) [inline README.md:3]: The branch should be mentioned as well so people can more easily find it

**Review by Axelen123 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# ⚠️ MOVING NOTICE ⚠️
```
Comment (oSumAtrIX) [inline README.md:1]: Only one heading is allowed. Move this under the title and use "##"

File: README.md (hunk 1 above)
Comment (oSumAtrIX) [reply README.md]: i think it makes more sense for this repo to redirect to the other, then the other can specify in which branch what happens

**Comment by BenjaminHalko** [comment]
@oSumAtrIX Formatting has been fixed

File: README.md [hunk 3]
```diff
@@ -6,6 +6,10 @@
 
 _(Yet another)_ rewrite of the ReVanced Manager using Kotlin and Jetpack Compose.
 
+## ⚠️ MOVING NOTICE ⚠️
```
Comment (oSumAtrIX) [inline README.md:9]:
```suggestion
## Notice
```

File: README.md [hunk 4]
```diff
@@ -6,6 +6,10 @@
...
+
+All future development on the Compose rewrite of ReVanced Manager has been moved to the [compose-dev](https://github.com/ReVanced/revanced-manager/tree/compose-dev) branch in the [revanced-manager](https://github.com/ReVanced/revanced-manager) repository.
```
Comment (oSumAtrIX) [inline README.md:11]:
```suggestion
Development is continued in the [ReVanced Manager](https://github.com/ReVanced/revanced-manager) repository.
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]
The emojis is unconventional in this readme.
The branch should be unspecified because it will change later and can be specified in the readme of the revanced manager repository but not this repository, because it will be archived.

**Comment by BenjaminHalko** [comment]
@oSumAtrIX done

**Comment by BenjaminHalko** [comment]
@oSumAtrIX done

File: README.md [hunk 5]
```diff
@@ -6,9 +6,9 @@
 
 _(Yet another)_ rewrite of the ReVanced Manager using Kotlin and Jetpack Compose.
 
-## ⚠️ MOVING NOTICE ⚠️
+## MOVING NOTICE
```
Comment (oSumAtrIX) [inline README.md]:
```suggestion
## Moved development
```

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**